src/subscription_manager/intentstore_interface.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function, division, absolute_import
+
+#
+# Copyright (c) 2018 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 2 (GPLv2). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+# along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+#
+# Red Hat trademarks are not licensed under GPLv2. No permission is
+# granted to use or replicate Red Hat trademarks that are incorporated
+# in this software or its documentation.
+#
+
+
+def set_intent_service_level(service_level):
```

Comment:
Add here doc string describing this function and arguments of this function.

```python
def set_intent_service_level(service_level):
    """
    This function ...
    """
```

This function can be also used for unsetting `servis_level`. The name of this function is little bit confusing. May be I would change the name of function or create another function for unsetting `service_level`. The decision will be up to you. If you will not change it, then please write about it in doc string.